dali/base/dautils.cpp

```diff
@@ -1016,6 +1026,11 @@ void CDfsLogicalFileName::setExternal(const char *location,const char *path)
     }
     StringBuffer str(EXTERNAL_SCOPE "::");
     str.append(location);
+    appendExternalStr(str,path);
+}
+
+void CDfsLogicalFileName::appendExternalStr(StringBuffer &str,const char *path)
```

Comment:
minor: as this method doesn't depend on anything in the object instance, it would make it clearer if it was declared as a static, and only passed the result back. Then setExternal and setPlaneExternal can call set() with that result.

"convertPosixPathToLfn" might be more intuitive.